ArduPlane/quadplane.cpp

```diff
@@ -3533,7 +3539,7 @@ float QuadPlane::forward_throttle_pct()
             return 0;
         } else {
             // calculate fwd throttle demand from manual input
-            float fwd_thr = rc_fwd_thr_ch->percent_input();
+            float fwd_thr = rc_fwd_thr_ch->norm_input()*100.0f;
```

Comment:
Can't just change to this, need be only if the new param is less than 0. The current `percent_input` ignores the trim position, `norm_input` does not so there is a change in behavior.